canonical-web-and-design/maas-ui
# feat(ui): add name column to machine network tab

## Done

- Add the name column for the machine interfaces on the network tab of machine details.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the network tab in machine details.
- You should see a list of interface names/mac addresses.

## Fixes

Fixes: #1959.
